package.json

```diff
@@ -36,7 +36,7 @@
     "test:debug": "mocha debug test/runner.js --reporter spec"
   },
   "dependencies": {
-    "broccoli-caching-writer": "^0.5.4",
+    "broccoli-caching-writer": "^1.1.0",
```

Comment:
Seems to work (see my comment in the source). Just have to add the `cacheInclude` option to limit when to rebuild.